### DISCUSSION TITLE
new Kitty terminal giving warning all time when try to copy paste using keyboard shortcuts

### DISCUSSION THREAD

**Discussion by ciscohack** [kovidgoyal/kitty#6818] [General]
Hello Experts,

In the new kitty terminal, I always get warnings when I try to copy/paste ...anyone know how to stop it as it's annoying?

<img width="1355" alt="image" src="https://github.com/kovidgoyal/kitty/assets/11479557/b3fc69ea-7041-4bd2-a7e9-678db5fc55dc">

**Comment by kovidgoyal** [comment]
Read the changelog it links you to the relevant option.

**Reply by ciscohack** [reply]
@kovidgoyal Thanks for the quick response. you mean the paste_actions right thanks

**Reply by agriffis** [reply]
How do you shut off the default `paste_actions` though? Is there a way to say "none"?

![image](https://github.com/kovidgoyal/kitty/assets/50637/10d4d63b-8b16-4319-bf0a-ad2fb1ab9035)

**Reply by kovidgoyal** [reply]
set it to filter and dont define a filter.

**Reply by xoores** [reply]
It is quite unpleasant to parse through an enormous changelog... Especially when distros add new versions of packages later and you don't really know *when* exactly that particular change was made :shrug: 

I personally found out by googling that string and sifting through sources (the only relevant result), finding `opts.paste_actions` that I could then Google again :sweat_smile:

Otherwise love Kitty, thanks for your efforts! :slightly_smiling_face: :+1:

**Comment by kovidgoyal** [comment]
yes

**Reply by ciscohack** [reply]
@kovidgoyal one more help, please. How i can chck or confirm that my kitty shell integration is working or not as I am not able to use the ctrl+shift+g option and many other dependent features

**Comment by ciscohack** [comment]
just copy paste this and job done in kitty.conf

paste_actions replace-dangerous-control-codes,replace-newline

**Reply by ifgar** [reply]
Did this but excluded `replace-newline` and works perfectly now.

**Comment by xoores** [comment]
What is the default that was in place before? I quite liked the previous behavior - that popup is *very* annoying, especially when configuring some switches when I frequently copy out parts and paste them as necessary :sweat_smile: 

Was it `paste_actions quote-urls-at-prompt` so the URLs are escaped and it does not fiddle around with anything unexpectedly?

Thanks!

**Comment by jeanlucthumm** [comment]
Still getting the pop-up:

![image](https://github.com/kovidgoyal/kitty/assets/4934853/5575fbda-509c-45ed-b4f3-1c55359bdf5d)


The following all did not work:

```
paste_actions quote-urls-at-prompt
```
```
paste_actions replace-dangerous-control-codes,replace-newline
```
```
paste_actions filter
```

**Reply by Quicksilver151** [reply]
same

**Comment by xoores** [comment]
Also having the same problem, any news regarding this please @kovidgoyal ?

**Comment by kovidgoyal** [comment]
There is no news. You cannot get those confirmation dialogs unless you have confirm in paste_actions. So dont put confirm in paste_actions.

**Comment by kesor** [comment]
Given the extreme annoyance of this popup. Would it be at least possible to provide information in the horrible popup about how to make it go away forever? Or will people must get annoyed the second time when trying to find some information about how to remove it, and there are absolutely zero hints in `man kitty.conf` regarding warnings, popups, etc...

**Comment by mturoci** [comment]
```
paste_actions no-op
```

did the trick for me

**Reply by priyanshu427** [reply]
works perfectly!

**Comment by GenesisOfTime** [comment]
Extra:

If you want to stop the same confirmation popups for reading the clipboard and modifying it, put this.

```
clipboard_control write-clipboard write-primary read-clipboard read-primary
```

**Reply by JellyApple102** [reply]
This is what did the trick for me, along with `paste_actions no-op`
